linkerd/app/outbound/src/lib.rs

```diff
@@ -153,10 +153,9 @@ impl<A: OrigDstAddr> Config<A> {
 
                 tcp_connect
                     .clone()
-                    // Initiates an HTTP client on the underlying transport.
-                    // Prior-knowledge HTTP/2 is typically used (i.e. when
-                    // communicating with other proxies); though HTTP/1.x fallback
-                    // is supported as needed.
+                    // Initiates an HTTP client on the underlying transport. Prior-knowledge HTTP/2
+                    // is typically used (i.e. when communicating with other proxies); though
+                    // HTTP/1.x fallback is supported as needed.
```

Comment:
it looks like all the comments in this file were re-wrapped...is this rustfmt's fault?

Reply:
I updated an editor setting that was inconsistent. I'd prefer 100char wrapping to 80.